src/main/java/seedu/address/ui/PersonViewPanel.java

```diff
@@ -35,47 +37,52 @@ public class PersonViewPanel extends UiPart<Region> {
 
     private CommandTextEditor commandTextEditor;
 
+    private final Image identificationImage = new Image(this.getClass()
+            .getResourceAsStream("/images/ui/identification-card.png"));
+    private final Image helplineImage = new Image(this.getClass().getResourceAsStream("/images/ui/help-line.png"));
+    private final Image pillImage = new Image(this.getClass().getResourceAsStream("/images/ui/pills.png"));
+    private final Image hospitalbedImage = new Image(this.getClass()
+            .getResourceAsStream("/images/ui/hospital-bed.png"));
+    private final Image calendarImage = new Image(this.getClass().getResourceAsStream("/images/ui/calendar.png"));
```

Comment:
`jpg` would probably be better in this case to help end users save space, but not a major concern.